pkg/debug/scripts/collect-info.sh

```diff
@@ -303,6 +303,22 @@ collect_kube_info()
     fi
 }
 
+collect_longhorn_info()
+{
+    type=$(cat /run/eve-hv-type)
+    if [ "$type" != "kubevirt" ]; then
+        return
+    fi
+    echo "- Collecting Longhorn specific info"
+    {
+        echo "  - longhorn support bundle "
+        # Longhorn issue reports have seen hangs generating this data 
+        # when nodes are down, or due to longhorn bugs.  
+        # Give up after 5min, and allow remaining system data to be collected.  
+        timeout 300s eve exec kube /usr/bin/longhorn-generate-support-bundle.sh
+    } > "$DIR/longhorn-info" 2>&1
```

Comment:
Can you paste the content of this file  $DIR/longhorn-info

Reply:
```
b5f9e667-c71a-491c-9395-bc12e376ab2c:/tmp/eve-info-v14-2024-02-14-23-02-21# cat longhorn-info 
  - longhorn support bundle 
Apply longhorn support bundle yaml at Wed Feb 14 23:02:33 UTC 2024
supportbundle.longhorn.io/support-bundle-collect-info created
support bundle progress percentage: 20
support bundle progress percentage: 20
support bundle progress percentage: 100
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 1002k  100 1002k    0     0   362M      0 --:--:-- --:--:-- --:--:--  489M
longhorn support bundle downloaded to /persist/newlog/kube/longhornsupportbundle_363a4509-6ee6-4da5-988f-7272fc094dbe_2024-02-14T23-02-35Z.zip
supportbundle.longhorn.io "support-bundle-collect-info" deleted
```

Reply:
@zedi-pramodh Is this output ok for review?

Reply:
looks ok